uefi-test-runner/src/boot/memory.rs

```diff
@@ -53,6 +51,44 @@ mod bootservices {
         }
         unsafe { boot::free_pool(ptr) }.unwrap();
     }
+
+    /// Tests getting the memory map.
+    pub fn test_memory_map() {
```

Comment:
Moving this function seems unrelated to the rest of the PR -- could you drop this change or move it to a separate PR?